blenderproc/python/types/MeshObjectUtility.py

```diff
@@ -516,12 +516,7 @@ def create_primitive(shape: str, **kwargs) -> "MeshObject":
     else:
         raise Exception("No such shape: " + shape)
 
-    primitive = MeshObject(bpy.context.object)
-    # Blender bug: Scale is ignored by default. #1060
-    if 'scale' in kwargs:
-        primitive.set_scale(kwargs['scale'])
-
-    return primitive
+    return MeshObject(bpy.context.object)
```

Comment:
I checked all primitives. Scale works for all except monkey and plane:

```suggestion
    primitive = MeshObject(bpy.context.object)
    # Blender bug: Scale is ignored by default for planes and monkeys. #1060
    if 'scale' in kwargs and shape in ["MONKEY", "PLANE"]:
        primitive.set_scale(kwargs['scale'])

    return primitive
```